nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -90,6 +90,15 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor MAX_ROWS_PER_FLOW_FILE = new PropertyDescriptor.Builder()
+            .name("Max Rows in Flow File")
+            .description("The maximum number of result rows that will be included in a single FlowFile. " +
```

Comment:
Good idea! I like it :) Lately we've been encouraging folks to add the "friendly" name (like "Max Rows in Flow File") using displayName() in the builder, and something like "qdbt-max-rows" as the name(), to help with internationalization. I realize the other properties don't follow this, and it's not a requirement, just mentioning it here to get the word out :)